yaml/landuse.yaml

```diff
@@ -1252,6 +1252,14 @@ filters:
       <<: *output_properties
       kind: dam
     extra_columns: [way]
+  - filter:
+      waterway: weir
+      geom_type: line
+    min_zoom: 12
```

Comment:
Why 12 and not 13 or 14? In your style when do you show it?

Reply:
Yes, it is used in our style which is similar to OpenTopoMap as OpenTopoMap shows this in lower zooms.